code/modules/research/xenobiology/crossbreeding/_clothing.dm

```diff
@@ -0,0 +1,143 @@
+	density = FALSE
+	anchored = TRUE
+	max_integrity = 10
+
+/obj/structure/light_prism/Initialize(mapload, var/newcolor)
+	. = ..()
+	color = newcolor
+	light_color = newcolor
+	set_light(5)
+
+/obj/structure/light_prism/attack_hand(mob/user)
+	to_chat(user, "<span class='notice'>You dispel [src]</span>")
+	qdel(src)
+
+/datum/action/item_action/change_prism_colour
+	name = "Adjust Prismatic Lens"
+	icon_icon = 'icons/obj/slimecrossing.dmi'
+	button_icon_state = "prismcolor"
+
+/datum/action/item_action/change_prism_colour/Trigger()
+	if(!IsAvailable())
+		return
+	var/obj/item/clothing/glasses/prism_glasses/glasses = target
+	var/new_color = input(owner, "Choose the lens color:", "Color change",glasses.glasses_color) as color|null
+	if(!new_color)
+		return
+	glasses.glasses_color = new_color
+
+/datum/action/item_action/place_light_prism
+	name = "Fabricate Light Prism"
+	icon_icon = 'icons/obj/slimecrossing.dmi'
+	button_icon_state = "lightprism"
+
+/datum/action/item_action/place_light_prism/Trigger()
+	if(!IsAvailable())
+		return
+	var/obj/item/clothing/glasses/prism_glasses/glasses = target
+	if(locate(/obj/structure/light_prism) in get_turf(owner))
+		to_chat(owner, "<span class='warning'>There isn't enough ambient energy to fabricate another light prism here.</span>")
+		return
+	if(istype(glasses))
+		if(!glasses.glasses_color)
+			to_chat(owner, "<span class='warning'>The lens is oddly opaque...</span>")
+			return
+		to_chat(owner, "<span class='notice'>You channel nearby light into a glowing, ethereal prism.</span>")
+		new /obj/structure/light_prism(get_turf(owner), glasses.glasses_color)
+
+/obj/item/clothing/head/peaceflower
+	name = "heroine bud"
+	desc = "An extremely addictive flower, full of peace magic."
+	icon = 'icons/obj/slimecrossing.dmi'
+	icon_state = "peaceflower"
+	item_state = "peaceflower"
+	slot_flags = ITEM_SLOT_HEAD
+	body_parts_covered = NONE
+	dynamic_hair_suffix = ""
+	force = 0
+	throwforce = 0
+	w_class = WEIGHT_CLASS_TINY
+	throw_speed = 1
+	throw_range = 3
+
+/obj/item/clothing/head/peaceflower/equipped(mob/living/carbon/human/user, slot)
+	. = ..()
+	if(slot == SLOT_HEAD)
+		user.add_trait(TRAIT_PACIFISM, "peaceflower_[REF(src)]")
+
+/obj/item/clothing/head/peaceflower/dropped(mob/living/carbon/human/user)
+	..()
+	user.remove_trait(TRAIT_PACIFISM, "peaceflower_[REF(src)]")
+
+/obj/item/clothing/head/peaceflower/attack_hand(mob/user)
+	if(iscarbon(user))
+		var/mob/living/carbon/C = user
+		if(src == C.head)
+			to_chat(user, "<span class='warning'>You feel at peace. <b style='color:pink'>Why would you want anything else?</b></span>")
+			return
+	return ..()
+
+/obj/item/clothing/suit/armor/heavy/adamantine
+	name = "adamantine armor"
+	desc = "A full suit of adamantine plate armor. Impressively resistant to damage, but weighs about as much as you do."
+	icon_state = "adamsuit"
+	item_state = "adamsuit"
+	flags_inv = NONE
+	obj_flags = IMMUTABLE_SLOW
+	slowdown = 4
+	var/hit_reflect_chance = 40
+
```

Comment:
Make its armor not that of DS gear

Reply:
I think the balancing factor is that while extremely durable, it also makes you about as slow as a snail and provides no head protection. It's also not innately space proof.

Reply:
Changed.